### ISSUE TITLE
Default browser on Win10 doesn't open links from 3rd parties

### ISSUE THREAD

**Issue by arisAlexis** [brave/brave-browser#3758]
## Description 
After making Brave the default browser on Windows 10, links from slack,discord stop opening links.

## Steps to Reproduce

   1. Make Brave default browser
   2. Open slack
   3. Follow link with click doesn't open

## Actual result:

Nothing
## Expected result:
Open a new window with the link

## Brave version (brave://version info)
Version 0.61.51 Chromium: 73.0.3683.75 (Official Build) (64-bit)

### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? 
Not sure

**Comment by rebron** [comment]
cc: @kjozwiak need more info.

**Comment by kjozwiak** [comment]
@arisAlexis I couldn't reproduce the issue you're experiencing using the following build under `Win 10 x64`:

```
Brave  | 0.61.51 Chromium: 73.0.3683.75 (Official Build) (64-bit)
--- | ---
Revision | 909ee014fcea6828f9a610e6716145bc0b3ebf4a-refs/branch-heads/3683@{#803}
OS | Windows 10 OS Build 17763.348
```

Does Brave appear as your default browser under Windows settings? What happens if you try setting Brave as the default browser again using `brave://settings`. Do you have multiple Brave installations?

**Comment by arisAlexis** [comment]
Brave is my default browser in windows and in the brave://setttings. The problem (which is not really so big because whatsapp opens links for example) started when I switched from Chrome to Brave. Slack and discord both don't open the links. Did you try with slack for example?

**Comment by kjozwiak** [comment]
@arisAlexis yup, I tried with both Slack & Discord and the links are opening in Brave which is set as the default browser. Are both `http` & `https` associated with Brave? You check by going into `Default Apps` --> `Set defaults by app` --> Click on `Brave` --> `Manage`. Quick example:

<img width="588" alt="Screen Shot 2019-03-21 at 12 54 25 AM" src="https://user-images.githubusercontent.com/2602313/54734128-f4174a00-4b73-11e9-9379-fd53d39389eb.png">

**Comment by arisAlexis** [comment]
yes both http/https were and I found a third app that does it, Pokerstars client. It doesn't find a default app and it needs manual reconfiguring. I think there is something there and it's quite a serious bug if the goal is to make users switch completely to Brave.

**Comment by rebron** [comment]
cc: @Brave-Matt this looks like we need to move to support and this might be a config issue. Anything you can think of here?

**Comment by abbynormative** [comment]
@rebron @Brave-Matt  I think I may have a related issue when Brave is set to default on a Mac. Info and steps below.

## Description 
When I receive a link over Signal Desktop and try to open it, Brave will open a new window for the link but not load the page. It only opens a blank browser window. I have to imagine this may also be an issue for other messaging clients.

## Steps to Reproduce
   1. Receive a link via Signal Desktop chat
   2. Click on the link in the Signal Desktop client
   3. A blank Brave browser will open but the page from the link doesn't load

## Actual result:
Blank Brave browser window

## Expected result:
A Brave browser window with the page from the Signal link loaded

## Reproduces how often: 
Every time.


## Brave version (brave://version info)

Brave | 0.63.48 Chromium: 74.0.3729.108 (Official Build) (64-bit)
Revision | daaff52abef89988bf2a26091062160b1482b108-refs/branch-heads/3729@{#901}
OS | Mac OS X

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
